packages/pangea-node-sdk/.gitlab-ci.yml

```diff
@@ -109,24 +109,6 @@ pangea-node-sdk-integration-tests-may-fail:
         TEST: file_scan
   allow_failure: true
```

Comment:
`file_scan` is now stable in GCP, we should remove its `allow_failure: true`

Reply:
https://github.com/pangeacyber/pangea-javascript/pull/892